examples/stage/odc_apps_k8s/efs-provisioner.tf.off

```diff
@@ -0,0 +1,111 @@
+resource "kubernetes_secret" "efs-provisioner" {
```

Comment:
We have use "_" in resource naming. Can we be consistent in this template too.

Reply:
I've committed this change but note the warning on the commit if you have existing infrastructure.